README.md

```diff
@@ -1,33 +1,32 @@
-# Project
+# Azure Terraform Verified Modules
 
-> This repo has been populated by an initial template to help get you started. Please
-> make sure to update the content to build a great experience for community-building.
+> This repo contains the source code of all currently available Terraform modules verified by Microsoft Azure. All verified modules will meet the quality pipeline established by Microsoft Azure team, which guarantee their high quality.    
 
-As the maintainer of this project, please make a few updates:
+## Modules
+Below is a table containing all Terraform modules verified by Microsoft Azure. Each module version badge shows the latest version of the corresponding module. You may click on a version badge to check all available versions for a specific module. Each minimum Terraform version shows the minimum entry requirements for the version of Terraform. Using the verified modules below the required Terraform version might cause potential inconsistency and disruption. 
+> For those with parallel modules, we strongly recommend you transfer your existing infrastructure to the new verified modules, as old parallel modules will no longer be updated. 
 
-- Improving this README.MD file to provide a great experience
-- Updating SUPPORT.MD with content about this project's support experience
-- Understanding the security reporting process in SECURITY.MD
-- Remove this section from the README
+<!-- Begin Module Table -->
+
+| Module            | Module Version | Minimum Terraform Version | Docs                                                 |Key Metrics                                                 | Version Details|
+| ----------        | :-----------:  | :-----------:             |----------                                            | :-----------:                                             |:-----------:   |
+| Terraform-AzureRM | 1.2.0          | 1.13.0                    |https://github.com/Azure/terraform-azurerm-subnets    | daily downloads     weekly downloads  yearly downloads    |docs            |
```

Comment:
This table is a bit too big for the view resulting in a horizontal scroll bar which is not ideal. Few suggested changes to prevent that:
1. Make the Module version number a hyper link to the version details.
2. Under docs provider a doc name and make it clickable. ie. [Readme.md](https://github.com/Azure/terraform-azurerm-subnets) instead of the full uri.
3. Can we have columns for each of the key metrics instead of lumping them into a single column?